library/alloc/src/ffi/mod.rs

```diff
@@ -89,3 +89,18 @@ pub use self::c_str::{FromVecWithNulError, IntoStringError, NulError};
 
 #[unstable(feature = "c_str_module", issue = "112134")]
 pub mod c_str;
+
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Presumably only the re-exports in libstd should have these, and the alloc/core originals should be marked unstable for now.

Reply:
Thanks, I've marked them as unstable for now.

Someone else interested in it could stablise it.